src/config.rs

```diff
@@ -155,3 +158,50 @@ create_config! {
     cfg_test: bool, true, false, "build cfg(test) code";
     unstable_features: bool, false, false, "enable unstable features";
 }
+
+/// A rustfmt config (typically specified via rustfmt.toml)
+pub struct FmtConfig(rustfmt::config::Config);
+
+impl FmtConfig {
+    /// Look for `.rustmt.toml` or `rustfmt.toml` in `path`, falling back
+    /// to the default config if neither exist
+    pub fn from(path: &Path) -> FmtConfig {
```

Comment:
Could we use the code in Rustfmt for this? Even if it means changing the API in Rustfmt a bit, I'd prefer that, rather than duplicating code, if we don't have to.